docs/gen_universe/gen_universe.markdown

```diff
@@ -0,0 +1,68 @@
+# Tutorial for using the gen_universe script
+
+The Universe repository provides tooling for generating your own Universe.
+
+Clone the Universe repository with:
+```bash
+git clone git@github.com:mesosphere/universe
+cd universe
+```
+
+Assuming that a package was created and saved in the following location:
+```bash
+tree ~/work/private-universe/
+```
```

Comment:
Why not merge both the code blocks OR add a text statement in between them if you feel user needs some help understanding.

Reply:
No good reason. One is a command and the other is the output. I wanted to make it easier for the user to copy and paste the command.